docs/src/diagrammer.js

```diff
@@ -0,0 +1,11 @@
+const parserFactory = require('grammar');
+const diagram = require('chevrotain/diagrams/src/main.js');
+
+
+const Parser = parserFactory.default('[Comment]', '[EndComment]').Parser;
+const parserInstanceToDraw = new Parser([]);
+
+
+module.exports = function(targetDiv) {
```

Comment:
Are you intentionally not activaiting the diagrams behavior(usage and definition coloring and page scrolling on none terminal click)?  If so maybe change the css style so none terminals won't have a clickable pointer cursor.